idr0016-wawer-bioactivecompoundprofiling/create-hdf

```diff
@@ -0,0 +1,220 @@
+#!/usr/bin/env python3
+
+from __future__ import print_function
+
+import csv
+import re
+import sys
+from pathlib import Path
+
+import h5py
+import numpy as np
+
+idr016_location = Path('/data/images/idr/idr016')
```

Comment:
`/uod/idr/**` is probably the closest we have to a standard location. Is the data currently under there?

Reply:
Yes, can be updated to use that.

Reply:
I think that would be good since it documents the source of the data even if you can't access it directly.

Reply:
Done.